### ISSUE TITLE
How to train model with linear layer  or conv layer?

### ISSUE THREAD

**Issue by feitiandemiaomi** [Sierkinhane/crnn_chinese_characters_rec#230]
**Thanks for great repo , The model can be trained normally, but when I replace the lstm layer with a convolutional layer or linear layer ,the model can't converge, and  the loss decreased from 270 to 4. I try to modify lr from 0.01 to 0.0000001, it also dosen't work, can you give some advice?**
![图片](https://user-images.githubusercontent.com/18325462/85695775-f23f1d80-b70a-11ea-8a00-47819a2811b3.png)
@Sierkinhane

**Comment by Sierkinhane** [comment]
If you use ctcloss, I advice you to check the version of pytorch, 1.2.0 is ok, other versions are not tested. Otherwise, maybe you should check whether  a nan existed in your tensor before computing loss.

**Comment by feitiandemiaomi** [comment]
**Yes,I use ctcloss , My verison is pytorch1.1.0 ,Just now ,I trained it with Pytorch1.2.0 ,the output are as follows:**
![图片](https://user-images.githubusercontent.com/18325462/85714651-36d3b480-b71d-11ea-9b37-3671055faab5.png)
**There was a problem from the beginning,Did you do something else, or could you provide the code?**

**Comment by Sierkinhane** [comment]
you can try this ctcloss implementation https://github.com/SeanNaren/Warp-ctc

**Comment by Sierkinhane** [comment]
@feitiandemiaomi
